test/webrtc.test.js

```diff
@@ -1,33 +1,275 @@
-import { WebSocket } from 'mock-socket';
+import 'regenerator-runtime/runtime';
 
 import {
   WEBRTC_PEER_CONFIG,
   WEBRTC_PEER_OPTIONS,
-  Logger
+  WEBRTC_JOIN_ROOM,
+  Logger, WEBRTC_INTERNAL_MESSAGE, WEBRTC_PEER_LEFT
```

Comment:
Put Logger as the last import so it's in order please :)

Reply:
OK :)